server/assets-v1/templates/src/pages/client_portal/login.html

```diff
@@ -163,22 +162,31 @@ <h1 class="font-bold text-4xl text-[#3751FE] mb-11 animate-slideFromLeft">Login<
             }
           );
           const responseTwo = await respTwo.json();
-          console.log("RESPONSE", responseTwo)
           if (responseTwo.token) {
             localStorage.setItem("clientToken", responseTwo.token);
-            alert("Login successful!");
+            showToastMessage("Login successful!", "success");
             window.location.href = "[[ .ProxyURL ]]/client-profile";
+          } else {
+            showToastMessage(responseTwo.message, "error");
```

Comment:
what if responseTwo doesnt have message field? i.e. 500 error

Reply:
What do you think If I add extra 
```
else if (responseTwo.message) {
   showToastMessage(responseTwo.message, "error");
} else {
   showToastMessage("Something went wrong!", "error");
}
```

Reply:
LGTM